ide/src/im.rs

```diff
@@ -1,36 +1,12 @@
-use std::{error::Error, fs::File, io::Read};
+use std::path::Path;
 
-pub fn get_logo(file_name: String) -> Result<(u32, u32, Vec<u8>), Box<dyn Error>> {
-    let mut bff = String::new();
+use game_engine::error::LoadError;
 
-    let mut file = File::open(&file_name).expect("Unable to open file");
-    file.read_to_string(&mut bff).expect("Unable to read");
-    let mut lines = bff.lines();
-    lines.next();
+pub fn get_logo<P: AsRef<Path>>(file_name: P) -> Result<(u32, u32, Vec<u8>), LoadError> {
+    let img = image::open(file_name)?;
+    let rgba = img.into_rgba8();
+    let (w, h) = rgba.dimensions();
+    let content = rgba.into_raw();
```

Comment:
Can someone check if this code works? I realize now that i have no way of seeing the window icon.

Also, since we're just parsing it to rgba anyways, do we have any reason to save `icon.ppm` as a ppm?

Reply:
This does not work. This is not the intended image Or maybe it works, but then the file format is wrong.
![image](https://user-images.githubusercontent.com/56885633/158690659-887b313c-b0a6-49b1-97db-73f5d186893b.png)